zen/pokedex/openrouter_sync.py

```
@@ -3,400 +3,410 @@
                 response.raise_for_status()
                 data = response.json()
-                
+
                 # Cache the response
-                self._save_cache(data['data'])
-                return data['data']
-                
+                self._save_cache(data["data"])
+                return data["data"]
+
             except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)